### PR TITLE
Style

### PR DESCRIPTION
**Pull request by NurkaAmre**
### PR is about:
- Used the details of the Bookstore design in 
- As much as possible followed style.

### CODE REVIEW

File: src/components/Book.js [hunk 1]
```diff
@@ -21,10 +21,12 @@ const Books = () => {
               id={book.id}
               title={book.title}
               author={book.author}
+              category={book.category}
             />
           ),
         )
       }
+      <div className="horizontal-line" />
```
Comment (danmainah) [inline src/components/Book.js:29]: - [ ] @NurkaAmre great work on your design, but some elements are not fitting well, and also the size of the percentage number is too big, kindly check the comparison below:

Zelpin Design          |  Your design     
:-------------------------:|:-------------------------:
image | image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @NurkaAmre , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

✔️ No linter errors
✔️ Github flow was correctly used
✔️  Good design


# Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by ha-manel — APPROVED** [review]
# STATUS: APPROVED ✅
## Hi @NurkaAmre 👋

Your project is complete! There is nothing else to say other than... it's time to merge it :shipit: 
**Congratulations! 🎉**



## Highlights:
* Well written PR description ✔️ 
* All tests passed ✔️ 

## Optional suggestions:
- [x] Kindly add the `MIT` file since you mentioned it in the README, because the link in your README is broken 👍 


------
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by NurkaAmre** [comment]
Thank you @ha-manel 😄

**Pull request merged**